### PR TITLE
Header

### PR DESCRIPTION
**Pull request by BBriset**
Static header

### CODE REVIEW

**Review by Quentin-Macquart — APPROVED** [review]
Long to read, but it's seems ok to me.

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import './Header.css';
+import React from 'react';
+import { bubble as Menu } from 'react-burger-menu';
+
+const Header = () => {
+  return (
+    <>
+      <Menu width="40%" className="menu">
```
Comment (Hacene010) [inline src/components/header/Header.jsx:8]: No style on the xml file :/

**Review by Hacene010 — CHANGES_REQUESTED** [review]
No css on the xml file

**Comment by BBriset** [comment]
not really css it's a props of the component from the node module, i don't find out another way to change this its fixed by default at 300 px... 

Width

You can specify the width of the menu with the width prop. The default is 300.

Menu.defaultProps = {
        bodyClassName: '',
        burgerBarClassName: '',
        burgerButtonClassName: '',
        className: '',
        crossButtonClassName: '',
        crossClassName: '',
        disableAutoFocus: false,
        disableCloseOnEsc: false,
        htmlClassName: '',
        id: '',
        itemClassName: '',
        itemListClassName: '',
        menuClassName: '',
        morphShapeClassName: '',
        noOverlay: false,
        noTransition: false,
        onStateChange: function onStateChange() {
        },
        outerContainerId: '',
        overlayClassName: '',
        pageWrapId: '',
        styles: {},
        width: 300,
        onIconHoverChange: function onIconHoverChange() {
        },
        itemListElement: 'nav'
    };
    return Menu;

**Comment by Hacene010** [comment]
Ok Bertrand I did not see it like that

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**